unused/Battle_of_Mobei.md

```diff
@@ -0,0 +1,5 @@
+The Battle of Mobei took place in 119 BC in the Northern Gobi Desert. It was fought between the Han and Xiognu, who at this point had been in a century-long conflict and fought a decade-long war. The Han army commanded by Wei Qing was much smaller and consisted of comparably inexperienced soldiers because another much more powerful Han army had been sent to invade Xiognu territory as well. And the plan was for only this other army to engage with the main Xiognu forces.
```

Comment:
```suggestion
The Battle of Mobei took place in 119 BC in the Northern Gobi Desert. It was fought between the Han and Xiognu, who at this point had been in a century-long conflict and fought a decade-long war. The Han army commanded by Wei Qing was much smaller and consisted of inexperienced soldiers because another more powerful Han army had been sent to invade Xiognu territory in a different location. And the plan was for only this other army to engage with the main Xiognu forces.
```